packages/callout/src/Callout.tsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import Icon, { glyphs } from '@leafygreen-ui/icon';
```

Comment:
We're beginning to move to importing icons directly from dist internally to improve the tree-shaking situation in users' apps since right now, the main files in our package.json point at a webpack-bundle, rather than an ES module 😞 

The pattern looks like:
```js
import Bulb from '@leafygreen-ui/icon/dist/Bulb;
```